WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Container/AddProductWithAIContainerView.swift

```diff
@@ -4,6 +4,7 @@ import SwiftUI
 final class AddProductWithAIContainerHostingController: UIHostingController<AddProductWithAIContainerView> {
     private let viewModel: AddProductWithAIContainerViewModel
     private var addProductFromImageCoordinator: AddProductFromImageCoordinator?
+    private var setToneAndVoiceBottomSheetPresenter: BottomSheetPresenter?
```

Comment:
We should remove this too.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10754/commits/c76c42ce24ec4590df78d47bc8f946b4ba92cdd1